frontend/index.html

```diff
@@ -100,6 +100,7 @@
   <!--Utils-->
   <script src="app/utils/utils.js"></script>
   <script src="app/utils/messageService.js"></script>
+  <script src="app/utils/EventListenerService.js"></script>
```

Comment:
Change the name of the pull request and description to the refact to user service to update the user, and remove the action to change current institution when the institution is removed  because if the removal is hierarchical and the user is a member of the parent institution and the daughter institution, when the parent institution is removed hierarchically, the change of the current institution of the user is changed to another institution also removed.